backend/process/src/socket/socket.js

```diff
@@ -87,28 +95,62 @@ export const registerHandlers = (io, socket) => {
     createTimer(game, ASKING_DURATION_S, null);
   };
 
-  const transitionToVoting = (game) => {
+  const transitionToVoting = async (game) => {
+    // Get question object
+    const questions = []
+    console.log("game in transition:")
+    console.log(game)
+
+    for (const id of game.questions) {
+      console.log("in send_answer getting question..")
+      console.log(id)
+      const docs = await getDocuments('questions', 'id', id)
+      console.log("docs:")
+      console.log(docs)
+      const question = docs[0]
+      
+      console.log(question)
+      questions.push(question)
+    }
+
+    console.log("question list retrieved:", questions)
     io.to(game.roomID).emit(
       "send_voteAnswers",
-      game.questions[game.round][game.questionIndex]
+      questions[game.round * 2 + game.questionIndex]
```

Comment:
nit: potentially replace the 2 here with some named constant (e.g. NUM_QUESTION_PER_ROUND)